app/api/resources/rapid_view.py

```diff
@@ -0,0 +1,63 @@
+from fastapi import APIRouter, Request, HTTPException
+from fastapi.responses import RedirectResponse
+import logging
+import re
+from core.logging import InterceptHandler
+from core.config import ENSEMBL_URL
+from api.utils.metadata import (
+    get_genome_id_from_assembly_accession_id,
+    get_assembly_accession_from_ncbi,
+)
+
+logging.getLogger().handlers = [InterceptHandler()]
+
+router = APIRouter()
+
+
+# Resolve species home
+# https://rapid.ensembl.org/Homo_sapiens_GCA_009914755.4/
+# https://rapid.ensembl.org/Homo_sapiens_GCA_009914755.4/Info/Index
+@router.get("/{species_url_name}/{subpath:path}", name="Rapid Species Home")
```

Comment:
Just checking: isn't this url pattern going to match everything within a species, including gene pages and so on? Are there plans to redirect to gene pages?

Reply:
yes there are plans to redirect. But I think we have to write the gene url or location url above this method. I think its the order that decides which method to run

Reply:
yes there are plans to redirect. But I think we will have to tackle this inside this method by checking the path for gene or location